docker/run_tests.sh

```diff
@@ -21,6 +21,11 @@ export LD_LIBRARY_PATH=${install_dir}/hdf5-${hdf5_version}/lib
 export LD_LIBRARY_PATH=${install_dir}/moab-${moab_version}/lib:${LD_LIBRARY_PATH}
 export LD_LIBRARY_PATH=${DAGMC_dir}/lib:${LD_LIBRARY_PATH}
 
+# test DAGMC CMake configuration file
+cd ${build_dir}/DAGMC-moab-${moab_version}/src/cmake/test_config
+cmake .
```

Comment:
Do we still need to provide some hint for where to find DAGMC?

Reply:
In CI the installed DAGMC location is added to the `LD_LIBRARY_PATH`. I've soft-linked FindDAGMC.cmake (that could be done differently) and it uses the hints there to find the DAGMC configuration.

Reply:
Should we also test a version that is not in LD_LIBRARY_PATH?  or does that not make sense?